teflo_datarouter_plugin/helpers.py

```diff
@@ -8,25 +8,25 @@
     :license: GPLv3, see LICENSE for more details.
 """
 import tarfile
-import os
 from teflo.exceptions import TefloReportError
 import shutil
 import os
+import requests
+import json
 
 
-def compose_pload(payload_dir, results_artufact_path, workspace):
+def compose_pload(payload_dir):
     tar_dir_list = []
     check_dir = validate_struc_before_compose(payload_dir)
     with tarfile.open(payload_dir + ".tar.gz", "w:gz") as ntar:
         ntar.add(payload_dir, arcname=os.path.basename(payload_dir))
         for member in ntar.getmembers():
             tar_dir_list.append(member.name)
-        shutil.copy2(f'{payload_dir}.tar.gz', f'{workspace}/{results_artufact_path}/')
         return tar_dir_list
 
 
 def validate_compose_payload_content(tar_dir_list):
-    pay_structer = ['/resultsdb', '/results', '/attachments']
+    pay_structer = ['resultsdb', 'results', 'attachments']
```

Comment:
this method will always fail if any one of the directory is not present. this should not be the case